sdk/endpoints/objects/v1/move_slo_request.go

```diff
@@ -22,10 +22,25 @@ type MoveSLOsRequest struct {
 }
 
 func (r MoveSLOsRequest) Validate() error {
-	return moveSLOsRequestValidation.Validate(r)
+	switch {
+	case r.IsMoveToService():
+		return moveSLOsToServiceRequestValidation.Validate(r)
+	default:
+		return moveSLOsToProjectRequestValidation.Validate(r)
+	}
 }
 
-var moveSLOsRequestValidation = govy.New[MoveSLOsRequest](
+// IsMoveToProject returns true if the request is for moving SLOs to another project.
+func (r MoveSLOsRequest) IsMoveToProject() bool {
+	return moveSLOsToProjectRequestValidation.Validate(r) == nil
+}
+
+// IsMoveToService returns true if the request is for moving SLOs to another service within the same project.
+func (r MoveSLOsRequest) IsMoveToService() bool {
+	return moveSLOsToServiceRequestValidation.Validate(r) == nil
+}
+
```

Comment:
please, not like this, why do you need to check it at the validation level in the first place?
why not just remove the `Required` condition from `newProject`?
then it will be up to the platform logic to decide what to do with this payload